### DISCUSSION TITLE
Physical Installation

### DISCUSSION THREAD

**Discussion by unintended-consequences** [revk/xESP32-Faikout#30] [Q&A]
Hi - I managed to source the board from amazon.co.uk, thank you! Any tips on physical installation? I'm using an older model - here is the PCB diagram. 
![image](https://user-images.githubusercontent.com/62865064/235049009-58a9425f-4916-4138-8d18-c1d75d072a2c.png)

Can I use the X33 connector? Is there a wiring diagram for the pinouts?

**Comment by unintended-consequences** [comment]
![image](https://user-images.githubusercontent.com/62865064/235049179-06bd8171-ae25-437c-8e90-034217f35328.png)

Here is the description of the connectors

**Comment by unintended-consequences** [comment]
![image](https://user-images.githubusercontent.com/62865064/235049245-44b6b649-b120-48d7-9c9e-514c8058259e.png)

This is listed under wiring diagram in the service manual but looks totally different.

**Reply by revk** [reply]
I believe it is S21 of X50.

Does that model work with the Daikin WiFi modules? If not? I suspect it will not work.

**Comment by unintended-consequences** [comment]
OK. I have some other models - but I realise they need a daughter board. e.g. I have a CTKS60QVM. Per this link: https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwiP2bzs_sv-AhVhwzgGHdulBVsQFnoECDYQAQ&url=https%3A%2F%2Fwww.lifestyleguru.com.sg%2Fimage%2Fcatalog%2FPage%25201%2520Daikin%2520Smart%2520Envi%2520single%2520split_2_Part1%2FCat%25203%2520Daikin%2520multi%2520split%25205_TICKS_SMIle%2520series%2520full%2520catalogue.pdf&usg=AOvVaw0W_ypmYe0IK7aVSjywL1Vb

![image](https://user-images.githubusercontent.com/62865064/235080516-74784c47-c2fd-4480-ad34-a9e857254c2f.png)
In order to connect to the Wifi adapter (BRP072A42), I need this adapter: KRP980B2. This adapter has the S21 connector onboard.

And this adapter seems to be around $250 USD locally, unfortunately.

**Reply by revk** [reply]
That's a bugger. I did think I made the listing clear, sorry.
